### PR TITLE
Better logs

### PR DESCRIPTION
**Pull request by edznux-dd**
This PR add some logs around error handling, fixes some small issues and add some text at the end of ingestion to point to the UI.

### CODE REVIEW

File: cmd/kubehound/root.go [hunk 1]
```diff
@@ -51,7 +51,18 @@ var (
...
 
-			return core.CoreLive(cobraCmd.Context(), khCfg)
+			err = core.CoreLive(cobraCmd.Context(), khCfg)
+			if err != nil {
+				return err
+			}
+
+			log.I.Warn("KubeHound as finished ingesting and building the graph successfully.")
+			log.I.Warn("Please visit the UI to view the graph by clicking the link below:")
+			log.I.Warnf("http://localhost:8888")
```
Comment (jt-dd) [inline cmd/kubehound/root.go:61]:
```suggestion
			log.I.Warn("http://localhost:8888")
```

File: cmd/kubehound/root.go [hunk 2]
```diff
@@ -51,7 +51,18 @@ var (
...
+			// Yes, we should change that :D
+			log.I.Warnf("Password being 'admin'")
```
Comment (jt-dd) [inline cmd/kubehound/root.go:63]:
```suggestion
			log.I.Warn("Password being 'admin'")
```

File: pkg/collector/k8s_api.go [hunk 3]
```diff
@@ -72,14 +72,17 @@ func NewK8sAPICollector(ctx context.Context, cfg *config.KubehoundConfig) (Colle
 	if err != nil {
 		return nil, err
 	}
+	if clusterName == "" {
+		return nil, errors.New("Cluster name is empty. Did you forget to set `KUBECONFIG` or use `kubectx` to select a cluster?")
+	}
```
Comment (jt-dd) [inline pkg/collector/k8s_api.go:77]: Did you ever got in this situation ? (wondering)

**Review by jt-dd — DISMISSED** [review]
nit

File: pkg/collector/k8s_api.go (hunk 3 above)
Comment (edznux-dd) [reply pkg/collector/k8s_api.go:77]: I did when testing a few thing starting from my personal PC that never connected to k8s.

**Review by jt-dd — APPROVED** [review]

**Pull request merged**